### ISSUE TITLE
Include anchors for items that people would want to reference

### ISSUE THREAD

**Issue by jsoref** [github/docs#7638]
### What article on docs.github.com is affected?

https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables

### What part(s) of the article would you like to see updated?

Each of the environment variables could have an `<a name=...` or `<a id=` so that I could link to a specific variable instead of just a page section and say "search for it yourself".

### Additional information

e.g. I'd love to be able to send a link like:

https://docs.github.com/en/actions/reference/environment-variables#GITHUB_BASE_REF

Basically, if you have a `<td><code>Foo</code></td>` it's probably helpful to everyone if there's a way to link to that by a pretty name (roughly `#Foo`, although possibly `#kindof-Foo` in case there are multiple cells of `Foo` and one of them is a `kindof` and another is, e.g. an `otherkind`).

**Comment by jsoref** [comment]
Example 2:
https://docs.github.com/en/graphql/reference/objects#user

I'd like to be able to link to `isLocked (...)` for `repositories (...)` for _something_ (for example User, although I think my original example is Organization):

  | `repositories` (`RepositoryConnection!`) |  
-- | -- | --

> `isLocked` (`Boolean`)
> If non-null, filters repositories according to whether they have been locked.

![image](https://user-images.githubusercontent.com/2119212/123314349-7bdd2300-d4f8-11eb-9526-60d9425a9472.png)

**Comment by janiceilene** [comment]
Thanks for the suggestion @jsoref! I'll triage this for the team to take a look 👀

**Comment by janiceilene** [comment]
Adding more anchors throughout the docs is definitely something that would be helpful and we'll look into! Since it needs to be done on a slower case by case basis, I'm going to close this issue in favor of several different internal issues where we're tracking potential updates.

Thanks for your continued interest in improving the GitHub docs ✨
